.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
### Your MIT file link
![mit-link-fixable](https://user-images.githubusercontent.com/93039359/186213904-dede8d8d-4d56-424e-96bb-931046cc2f2f.png)

### Expected MIT file link
![fixed](https://user-images.githubusercontent.com/93039359/186213935-e051ca69-fb60-4e35-a5d6-89f3232cd5a6.png)

- [ ] Please change your MIT file link in your readme as shown in the image above. This is the link to your license file of this project so please take it seriously :thinking: